boxsdk/object/storage_policy.py

```diff
@@ -0,0 +1,39 @@
+# coding:utf-8
+
+from __future__ import unicode_literals, absolute_import
+
+import json
+
+from .base_object import BaseObject
+from boxsdk.util.translator import Translator
+
+
+class StoragePolicy(BaseObject):
+    """Represents the storage policy"""
+
+    def get_url(self, *args):
+        """
+        Get url for storage policies.
+        """
+        return self._session.get_url('storage_policies', self._object_id, *args)
+
+    _item_type = 'storage_policy'
```

Comment:
Maybe nicer to have _item_type at the top of the class, similar to other object classes?